### DISCUSSION TITLE
Interested in Tooling Maintainership

### DISCUSSION THREAD

**Discussion by fg-j** [paketo-buildpacks/feedback#164] [Tooling Team]
Hi  Tooling Maintainers. I'm interested in becoming a maintainer on this subteam! I have a track record of contributing to the github-config and packit repos that includes:

- Enabling `jam update-buildpack` to work when the registry location of a buildpackage doesn't match its buildpack ID -- and preventing `jam update-buildpack` and `jam update-builder` from pulling prerelease versions.
- Adding workflows and scripts shared across the Paketo Builder repos
- Adding commit-signing and signed-commit-checking to the PR auto-merge workflows

The tooling repos can be a process bottleneck because so many other repos/buildpacks/contributors rely on them. I hope to take on some of the PR review and triage burden.

**Comment by ForestEckhardt** [comment]
@ryanmoran It appears that there is a super majority of approval is this enough to promote @fg-j?

**Comment by ryanmoran** [comment]
Yep. @fg-j congrats on your promotion to Tooling Maintainer! 🎉
